docs/fiddles/menus/customize-menus/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Customize Menus</title>
+  </head>
+
+  <body>
+    <div class="section-wrapper">
+      <h1>Customize Menus</h1>
+
+      <h3>
+        The <code>Menu</code> and <code>MenuItem</code> modules can be used to
+        create custom native menus.
+      </h3>
+
+      <p>
+        There are two kinds of menus: the application (top) menu and context
+        (right-click) menu.
+      </p>
+
+      <p>
+        Open the
+        <a href="http://electron.atom.io/docs/api/menu"
+          >full API documentation<span class="u-visible-to-screen-reader"
+            >(opens in new window)</span
+          ></a
+        >
+        in your browser.
+      </p>
+    </div>
+

```

Comment:
Since there are two sections here maybe add a header here:
```suggestion
<h2>Create an application menu</h2>
```